### ISSUE TITLE
non verbose fibex containing apid but not ctid for a message are ignored

### ISSUE THREAD

**Issue by mbehr1** [mbehr1/adlt#116]
There seem to be non-verbose fibex files where ctids are not provides for a message but apids are.
This is a bit weird as the dlt extended header has either no apid/ctid or both of them.

These cases should be mapped to e.g. a default ctid ('' or 'none' or '<na>' or to the apid as ctid) so that at least the apid info doesn't get lost.

**Comment by mbehr1** [comment]
Here is an example:
```
    <fx:FRAME ID="ID_10881">
            <ho:SHORT-NAME>ID_10881</ho:SHORT-NAME>
            <fx:BYTE-LENGTH>0</fx:BYTE-LENGTH>
            <fx:FRAME-TYPE>OTHER</fx:FRAME-TYPE>
            <fx:PDU-INSTANCES>
               <fx:PDU-INSTANCE ID="P_10881_0">
                  <fx:PDU-REF ID-REF="PDU_ID_10881_0"/>
                  <fx:SEQUENCE-NUMBER>0</fx:SEQUENCE-NUMBER>
               </fx:PDU-INSTANCE>
            </fx:PDU-INSTANCES>
            <fx:MANUFACTURER-EXTENSION>
               <MESSAGE_TYPE>DLT_TYPE_LOG</MESSAGE_TYPE>
               <MESSAGE_INFO>DLT_LOG_INFO</MESSAGE_INFO>
               <APPLICATION_ID>SYST</APPLICATION_ID>
               <CONTEXT_ID/>
               <MESSAGE_SOURCE_FILE>mcu/source/app/IO_Lifecycle/PWR/B1_920/pwr_B1_920.c</MESSAGE_SOURCE_FILE>
               <MESSAGE_LINE_NUMBER>357</MESSAGE_LINE_NUMBER>
            </fx:MANUFACTURER-EXTENSION>
         </fx:FRAME>
```

**Comment by mbehr1** [comment]
fixed with PR#117 with '' ctid
